CONTRIBUTING.md

```diff
@@ -19,11 +19,12 @@ Setup guides, FAQ, the more information we have on the [wiki](https://wiki.serva
 
 1. Fork Readarr
 2. Clone the repository into your development machine. [*info*](https://help.github.com/articles/working-with-repositories)
-3. Install the required Node Packages `yarn install`
-4. Start gulp to monitor your dev environment for any changes that need post processing using `yarn start` command.
-5. Build the project in Visual Studio, Setting startup project to `Readarr.Console` and framework to `netcoreapp31`
-6. Debug the project in Visual Studio
-7. Open http://localhost:8787
+3. Grab the submodules `git submodule init && git submodule update`
```

Comment:
Ditch this, no submodules

Reply:
drop this whole file; it's already updated

Reply:
copied your latest file baker and inserted that